ds13/code/modules/mob/living/psychosis_new.dm

```diff
@@ -0,0 +1,106 @@
+#define PSYCHOSIS_EFFECT_LAUGHTER 1
+#define PSYCHOSIS_EFFECT_HITSELF 2
+#define PSYCHOSIS_EFFECT_TRIPPY 3
+#define PSYCHOSIS_EFFECT_WEIRDSCREEN 4
+
+/datum/extension/psychosis
+	name = "Psychotic trauma"
+	expected_type = /mob/living/carbon/human
```

Comment:
is there a need to restrict this to humans? making it mob/living may be useful for future